tests/unit/test_session.py

```diff
@@ -875,6 +875,24 @@ def test_endpoint_from_production_variants_with_accelerator_type(sagemaker_sessi
         Tags=tags)
 
 
+def test_update_endpoint_succeed(sagemaker_session):
+    ims = sagemaker_session
+    ims.sagemaker_client.describe_endpoint = Mock(return_value={'EndpointStatus': 'InService'})
+    endpoint_name = "some-endpoint"
+    endpoint_config = "some-endpoint-config"
+    returned_endpoint_name = sagemaker_session.update_endpoint(endpoint_name, endpoint_config)
+    assert returned_endpoint_name == endpoint_name
+
+
+def test_update_endpoint_non_existing_endpoint(sagemaker_session):
+    ims = sagemaker_session
+    error = ClientError({'Error': {'Code': 'ValidationException', 'Message': 'Could not find entity'}}, 'foo')
+    error_message = 'Endpoint with name "non-existing-endpoint" does not exist; please use an existing endpoint name'
```

Comment:
Is this error message necessary? I don't see any assertion about it.

Reply:
Add the error message to pytest.raises(ValueError, message=error_message)

Reply:
Got it, I got confused with the order of statements. nit: rename it to `expected_error_msg` or something like that.

Reply:
Done. Also changed the naming in test_local_session and test_mxnet_train.